### ISSUE TITLE
Switch csv for compliance to conform to cointrackr's standards.

### ISSUE THREAD

**Issue by mark-antony1** [needmoney90/merkato#217]
The current format for the tax audits in csv format is currently not compliant with cointrackr, https://www.cointracker.io. I would recommend we use this platform for tax audits, as they have been the most recommended platform from accountants I have spoken to within the space. 

To conform to the cointrackr standard, we should do the following.
<img width="791" alt="screen shot 2018-06-21 at 1 56 22 pm" src="https://user-images.githubusercontent.com/18706329/41746101-09dd29c8-755e-11e8-9e86-6a30b6bbb4b9.png">

**Comment by nasaWelder** [comment]
Those are the only columns they want? Not enough information in a tax audit situation. There's no order ID or exchange name. Does it error if there are extra columns?

**Comment by mark-antony1** [comment]
Not sure about extra columns. It is enough, the dates are needed not tx id

**Comment by nasaWelder** [comment]
I agree it's enough for calculating taxes, I don't want to throw out that metadata, for audit situation

**Comment by 15chrjef** [comment]
Thats fine

**Comment by mark-antony1** [comment]
We can have two csv outputs if you like, but cointracker should take care of all concerns with audits.
